lnwallet/wallet.go

```diff
@@ -431,6 +431,56 @@ func (l *LightningWallet) Shutdown() error {
 	return nil
 }
 
+// PublishTransaction wraps the wallet controller tx publish method with an
+// extra rebroadcaster layer if the sub-system is configured.
+func (b *LightningWallet) PublishTransaction(tx *wire.MsgTx, label string) error {
```

Comment:
Where is this method used?

Reply:
So this will intercept the `PublishTransaction` call to the underlying `WalletController` (which is embedded).